depends/README.md

```diff
@@ -1,24 +1,107 @@
-### Usage
+# Depends build
+
+This is a system of building and caching dependencies necessary for building
+Bitcoin Core. It supports cross-compilation. For more details see [description.md](description.md),
+as well as [packages.md](packages.md) for how to add packages.
+
+## Usage
+
+### Ubuntu & Debian
+
+    apt install cmake curl make patch
+
+Skip the following packages if you don't intend to use the GUI and will build with [`NO_QT=1`](#dependency-options):
+
+    apt install bison g++ ninja-build pkgconf python3 xz-utils
 
 To build dependencies for the current arch+OS:
 
     make
 
-To build for another arch/OS:
+### macOS
 
-    make HOST=host-platform-triplet
+Install Xcode Command Line Tools and Homebrew Package Manager,
+see [build-osx.md](../doc/build-osx.md).
 
-For example:
+    brew install cmake make ninja
 
-    make HOST=x86_64-w64-mingw32 -j4
+To build dependencies for the current arch+OS:
+
+    gmake
+
+### FreeBSD
+
+    pkg install bash
+
+To build dependencies for the current arch+OS:
+
+    gmake
+
+### NetBSD
+
+    pkgin install bash gmake
+
+To build dependencies for the current arch+OS:
+
+    gmake
+
+### OpenBSD
+
+    pkg_add bash gmake gtar
+
+To build dependencies for the current arch+OS:
+
+    gmake
+
+## Configuring Bitcoin Core
 
 **When configuring Bitcoin Core, CMake by default will ignore the depends output.** In
 order for it to pick up libraries, tools, and settings from the depends build,
 you must specify the toolchain file.
-In the above example, a file named `depends/x86_64-w64-mingw32/toolchain.cmake` will be
+In the above example for macOS, a file named `depends/aarch64-apple-darwin24.4.0/toolchain.cmake` will be
```

Comment:
Please don't add triples with trailing versions numbers, as it's just something else to be "bumped". You can leave the windows example as-is.

Reply:
There's no Windows example above, but I'll switch it to Linux - which doesn't use a version.

Reply:
https://github.com/bitcoin/bitcoin/compare/7238f242d78f3d71834764031d7904d19525bab3..22cff32319de64cb98e1c89b9a7ed35611e89e27

Reply:
Not sure what you mean. There's no need to change anything here, the Windows example was fine before your PR, and is still fine now.

Reply:
The commit message of 99e6490dc51adde35b58e8d193aca7c1c422dbf3 explains why:

> In the Configuring section, use Linux native compilation as the
> example instead of Windows cross-compile.

Previously this documented started with cross compilation, which "above" referred to. In this PR I start with native compilation.